pages/en-us/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import { NextPage } from 'next'
+import NextLink from 'next/link'
+import { useTheme, Button, Card, Grid } from 'components'
+import PackageIcon from '@geist-ui/react-icons/package'
+import FeatherIcon from '@geist-ui/react-icons/feather'
+import GitHubIcon from '@geist-ui/react-icons/github'
+
+const Application: NextPage<{}> = () => {
+  const theme = useTheme()
+
+  return (
+    <>
+      <div className="layout">
+        <div className="hero">
+          <h1 className="hero__title">Geist</h1>
+          <h3 className="hero__description">
+            An open source design system for building modern websites and applications.
+          </h3>
+          <div className="hero__actions">
+            <NextLink href="/en-us/guide/installation">
+              <Button auto type="success" shadow scale={1.2}>
+                Documentation
```

Comment:
The current user is at the documentation site, where the characters are still `documentation` is a bit strange, or change to `Quick Start` would be better.